documentation/modules/auxiliary/admin/networking/ubiquiti_config.md

```diff
@@ -37,7 +37,7 @@
 
 ### Unf File
 ```
-resource (unifi_config.rb)> use auxiliary/admin/ubiquiti/ubiquiti_config
+resource (unifi_config.rb)> use auxiliary/admin/networking/ubiquiti_config
```

Comment:
Is this resource prompt needed? Or should this not be msf's default prompt followed by the module specific prompt? I don't see any references to loading up a resource file here so I'm confused how and why the documentation shows the prompt as `resource (unifi_config.rb)` as you don't seem to be loading a resource. This applies to all prompts in this documentation.